sdk/sdk-core/aar/src/main/java/mil/emp3/core/editors/milstd/MilStdDCSectorRangeFanEditor.java

```diff
@@ -273,7 +281,12 @@ protected void assembleControlPoints() {
 
         // Now add the range CPs.
         for (int index = 0; index < this.rangeList.size(); index++) {
-            this.createRangeAndAzimuths(index);
+            this.createRangeAndAzimuths(index, cpList);
+
+            while (cpList.size() > 0) {
+                this.addControlPoint(cpList.get(0));
```

Comment:
IMO we could write `this.addControlPoint(cpList.remove(0));` because `List.remove(index)` returns the element at the specified position.